.github/workflows/remove-wheels.yml

```diff
@@ -0,0 +1,81 @@
+name: Remove old wheels
+
+on:
+  # Run daily at 1:23 UTC
+  schedule:
+  - cron:  '23 1 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# Needed for micromamba pickup
+defaults:
+  run:
+    shell: bash -l {0}
+
+env:
+  N_LATEST_UPLOADS: 5
+  ANACONDA_USER: "scientific-python-nightly-wheels"
+
+jobs:
+  remove:
+
+    runs-on: ubuntu-latest
+    # Set required workflow secrets in the environment for additional security
+    # https://github.com/scientific-python/upload-nightly-action/settings/environments
+    environment:
+      name: remove-old-wheels
+
+    steps:
+      - name: Install micromamba and anaconda-client
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-name: remove-wheels
+          create-args: >-
+            python=3.11
+            anaconda-client=1.11.2
+
+      - name: Show environment
+        run: env
+
+      - name: Show CLI API info
+        run: |
+          anaconda show --help
+          echo ""
+          anaconda remove --help
+
+      - name: Query package index for packages
+        run: |
+          anaconda show "${ANACONDA_USER}" &> >(grep "${ANACONDA_USER}/") | \
+              awk '{print $1}' | \
+              sed 's|.*/||g' > package-names.txt
+
+      - name: Remove old uploads to save space
+        run: |
+          # Remove all _but_ the last ${N_LATEST_UPLOADS} package versions
+          # N.B.: `anaconda show` places the newest packages at the bottom of the output
+          # of the 'Versions' section and package versions are preceded with a '   + '.
+
+          if [ -s package-names.txt ]; then
+              # Remember can't quote subshell as need to split on (space seperated) token
+              for package_name in $(cat package-names.txt); do
```

Comment:
@stefanv @bsipocz this uses the `cat` approach that you both found easier to read, but I've added this comment above it as if people quote subshells without thinking (which happens a lot in my experience) they will get a bug. Short example:

```console
$ echo -e "name1 abc\nname2" > /tmp/example.txt
$ cat /tmp/example.txt 
name1 abc
name2
$ while IFS= read -r line; do echo -e "line: ${line}"; done </tmp/example.txt  # desired behavior
line: name1 abc
line: name2
$ for line in "$(cat /tmp/example.txt)"; do echo -e "line: ${line}"; done  # undesired behavior
line: name1 abc
name2
$ for line in $(cat /tmp/example.txt); do echo -e "line: ${line}"; done  # potentially desired behavior
line: name1
line: abc
line: name2
```

This example adds in a factor of space separated tokens which doesn't appear in our particular use case to show the general example of how using `read` when you want _per line_ splitting helps protect you.